samples/samples.js

```diff
@@ -117,6 +117,18 @@
 			path: 'scales/time/combo.html'
 		}]
 	}, {
+		title: 'TimeSeries scale',
```

Comment:
I'm not sure we need to duplicate the 3 examples, maybe 1 is enough and could be added to the `Time Scale` group at the end (`title: Series (TimeSeries Scale)`).